spec/acceptance/message_actions_spec.rb

```diff
@@ -3,97 +3,98 @@
 describe "Actions for Messages", :sqs do
 
   before do
-    sqs.queues.create("test")
+    sqs.create_queue(queue_name: "test")
   end
 
-  let(:sqs) { AWS::SQS.new }
-  let(:queue) { sqs.queues.named("test") }
+  let(:sqs) { Aws::SQS::Client.new }
+  let(:resource) { Aws::SQS::Resource.new }
+  let(:queue) { resource.get_queue_by_name(queue_name: "test") }
 
   specify "SendMessage" do
     msg = "this is my message"
-    result = queue.send_message(msg)
-    result.md5.should eq Digest::MD5.hexdigest(msg)
+    result = queue.send_message(message_body: msg)
+    result.md5_of_message_body.should eq Digest::MD5.hexdigest(msg)
   end
 
   specify "ReceiveMessage" do
     body = "test 123"
-    queue.send_message(body)
-    message = queue.receive_message
-    message.body.should eq body
+    queue.send_message(message_body: body)
+    messages = queue.receive_messages
+    messages.first.body.should eq body
   end
 
   specify "DeleteMessage" do
-    queue.send_message("test")
+    queue.send_message(message_body: "test")
 
-    message1 = queue.receive_message
-    message1.delete
+    messages = queue.receive_messages
+    messages.first.delete
 
     let_messages_in_flight_expire
 
-    message2 = queue.receive_message
-    message2.should be_nil
+    messages = queue.receive_messages
+    messages.first.should be_nil
   end
 
   specify "DeleteMessageBatch" do
-    queue.send_message("test1")
-    queue.send_message("test2")
+    queue.send_message(message_body: "test1")
+    queue.send_message(message_body: "test2")
 
-    message1 = queue.receive_message
-    message2 = queue.receive_message
-    queue.batch_delete(message1, message2)
+    queue.receive_messages.batch_delete!
```

Comment:
This is one of the specs still failing. The batch delete appears to delete one of the messages, but not the second. Any ideas why?